packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -55,6 +57,12 @@ export class Combobox extends FormAssociatedCombobox {
     @attr({ attribute: "autocomplete", mode: "fromView" })
     autocomplete: ComboboxAutocomplete | undefined;
 
+    /**
+     * @internal
+     */
+    @observable
+    public slottedLabel: Node[];
```

Comment:
I notice that we sometimes cast slotted elements as `HTMLElement[]` as in button:
 `public defaultSlottedContent: HTMLElement[];`

Here we cast as `Node[]`.  We should probably be consistent, which is the best choice?

Reply:
I think it depends. This is copy/paste and likely wrong. I think here we probably need HTMLElement because there isn’t really a way to project text here as it will require a shot attribute. However, in checkbox and radio, those likely could be text and therefore I think Node is probably correct.